src/app/shared/pipes/order-by.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Courses } from 'src/app/models/course.interface';
+
+@Pipe({
+  name: 'orderBy'
+})
+export class OrderByPipe implements PipeTransform {
+
+  transform(course: Courses[]): Courses[] {
+    const sortByOrder = [...course];
+    return sortByOrder.sort((currentElement, previousElement) => {
+      const dataA = +new Date(currentElement.creationDate);
```

Comment:
move this callback to separate method or give it appropriate name, It's hard to get how you sort here.
`sortByOrder.sort(this.newCoursesFirst)` something like that

Reply:
done